reddit/reddit.go

```diff
@@ -109,3 +112,43 @@ func MaxFeedForCtx(ctx context.Context) int {
 
 	return GuildMaxFeedsNormal
 }
+
+func CheckSubreddit(data string) (bool, error) {
```

Comment:
`name` seems like it describes this parameter more appropriately.
```suggestion
func CheckSubreddit(name string) (bool, error) {
```
...and ditto for usages.